recipes/vcftools/0.1.11/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
 
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+sed -i -e 's/DIRS = cpp perl/DIRS = cpp/' ./Makefile
```

Comment:
For macOS, this needs to be `sed -i.bak` (that is, the `-i` needs a backup extension)